docker/docker-compose.yml

```diff
@@ -150,7 +150,7 @@ services:
     ports:
       - '1433:1433'
     healthcheck:
-      test: ['CMD', '/opt/mssql-tools/bin/sqlcmd', '-Usa', '-PPr1sm4_Pr1sm4', '-Q', 'select 1']
+      test: ['CMD', '/opt/mssql-tools18/bin/sqlcmd', '-C', '-Usa', '-PPr1sm4_Pr1sm4', '-Q', 'select 1']
```

Comment:
Needed to implicitly trust the server certificate without validation. https://learn.microsoft.com/en-us/sql/tools/sqlcmd/sqlcmd-utility?view=sql-server-ver16&tabs=go%2Cwindows&pivots=cs1-bash#-c